src/Microsoft.Identity.Web/TokenCacheProviders/MsalAbstractTokenCacheProvider.cs

```diff
@@ -17,8 +17,7 @@ public abstract class MsalAbstractTokenCacheProvider : IMsalTokenCacheProvider
         /// Initializes the token cache serialization.
         /// </summary>
         /// <param name="tokenCache">Token cache to serialize/deserialize.</param>
-        /// <returns>A <see cref="Task"/> that represents a completed initialization operation.</returns>
-        public Task InitializeAsync(ITokenCache tokenCache)
+        public void Initialize(ITokenCache tokenCache)
```

Comment:
This is technically a breaking change. Is there anything special you need to do to rev the library version?

Reply:
we are going w/henrik's suggestion and doing obsolete attributes so we don't have to take a major version right now.